swagger/src/main/scala/org/http4s/rho/swagger/SwaggerModelsBuilder.scala

```diff
@@ -157,6 +157,27 @@ private[swagger] class SwaggerModelsBuilder(formats: SwaggerFormats) {
     linearizeStack(rr.path::Nil).flatMap(go(_, None)).headOption
   }
 
+  def collectSecurityScopes(rr: RhoRoute[_]): List[Map[String, List[String]]] = {
+
+    def go(stack: List[PathOperation], summary: Option[Map[String, List[String]]]): Option[Map[String, List[String]]] =
+      stack match {
+        case PathMatch("")::Nil                => go(Nil, summary)
+        case PathMatch(s)::xs                  => go(xs, summary)
+        case PathCapture(id, _, parser, _)::xs => go(xs, summary)
+        case CaptureTail::xs                   => summary
+
+        case MetaCons(_, meta)::xs =>
```

Comment:
Could this be dramatically by matching against
```
case MetaCons(_, RouteSecurityScope(secScope))::xs => .. whatever ..
case _::xs => go(xs, summary)
case Nil => summary
```

Reply:
Good point. I just copied from `collectSummary` without daring to touch/change too much.
Fixed now